examples/python3/Dockerfile

```diff
@@ -1,8 +1,14 @@
-FROM python:3-slim AS build-env
+FROM python:3.6-stretch AS build-env
 ADD . /app
 WORKDIR /app
+RUN pip3 install --upgrade pip
```

Comment:
Would it make sense to run `python -m compileall /app` too?

Reply:
That could be an option. I don't want to distract the user too much but I do see your point.